cli/src/main/java/com/box/l10n/mojito/cli/command/DropExportCommand.java

```diff
@@ -132,24 +140,48 @@ public void execute() throws CommandException {
      * @throws CommandException
      */
     List<String> getBcp47TagsForExportFromParam(Repository repository) throws CommandException {
-
+        
         List<String> bcp47tags = new ArrayList<>();
         
         List<String> validTags = new ArrayList<>();
-
+        
         for (RepositoryLocale repositoryLocale : repository.getRepositoryLocales()) {
             validTags.add(repositoryLocale.getLocale().getBcp47Tag());
         }
-
+        
         if (validTags.containsAll(bcp47tagsParam)) {
             bcp47tags.addAll(bcp47tagsParam);
         } else {
             bcp47tagsParam.removeAll(validTags);
             String invalidLocales = StringUtils.collectionToDelimitedString(bcp47tagsParam, ", ");
             throw new CommandException("Locales [" + invalidLocales + "] do not exist in the repository");
         }
-
+        
         return bcp47tags;
     }
-
+    
+    private boolean shouldCreateDrop(Repository repository) {
+        boolean createDrop = false;
+        Set<String> localesforTranslation = getLocalesForTranslation(repository);
+        RepositoryStatistic repoStat = repository.getRepositoryStatistic();
+        if (repoStat != null) {
+            for (RepositoryLocaleStatistic repoLocaleStat : repoStat.getRepositoryLocaleStatistics()) {
+                if (localesforTranslation.contains(repoLocaleStat.getLocale().getBcp47Tag()) && repoLocaleStat.getForTranslationCount() > 0L) {
+                    return true;
+                }
+            }
+        }
+        return createDrop;
+    }
+    
+    private Set<String> getLocalesForTranslation(Repository repository) {
+        Set<String> localesForTranslation = new HashSet<>();
```

Comment:
isn't it the same logic as `getBcp47TagsForExportFromRepository()` maybe just have `Sets.newHashSet(getBcp47TagsForExportFromRepository(repository));` ?